src/sentry/integrations/slack/views/unlink_team.py

```diff
@@ -86,21 +118,34 @@ def handle(self, request: Request, signed_params: str) -> HttpResponse:
             external_id=channel_id,
         )
         if len(external_teams) == 0:
+            _logger.info(
+                "no-team-found",
+                extra={logger_params},
```

Comment:
should this be just logger_params? i think it's already a dict

Reply:
Yeah, I think this would make it a singleton set.